File: stacks.js
class Stack {
  constructor() {
    this.stack = []
  }

  push(item) {
    this.stack.push(item)
    return 'Item has been added to the top'
  }

  pop(item) {
    this.stack.pop()
    return 'Item has been popped off the top'
  }

  peek() {
    return this.stack[this.stack.length - 1]
  }
}

firstStack = new Stack()

console.log(firstStack.push(1))
console.log(firstStack.push(2))
console.log(firstStack.peek())
console.log(firstStack.push(3))
console.log(firstStack.peek())


console.log(firstStack.pop())
console.log(firstStack.pop())

console.log(firstStack.stack)

File: stacks.py
class Stack:
    def __init__(self):
        self.stack = []

    def push(self, item):
        self.stack.append(item)
        return 'Item has been added to the top'

    def pop(self):
        self.stack.pop()
        return 'Item has been popped off the top'

    def peek(self):
        return self.stack[-1]


first_stack = Stack()

print(first_stack.push(1))
print(first_stack.push(2))
print(first_stack.peek())
print(first_stack.push(3))
print(first_stack.peek())


print(first_stack.pop())
print(first_stack.pop())

print(first_stack.stack)
